### PR TITLE
Project Proposal

### PR DESCRIPTION
**Pull request by eenagy**
@taxmeifyoucan Here is my project proposal.

### CODE REVIEW

File: projects/eth-deb-repository.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Running `apt install geth-mainnet-full` would seamlessly install all necessary software components, such as the go-ethereum client, while configuring a default full node mainnet setup and initiating it as a service for immediate usability.
+- Running `apt install lighthouse` would likewise install all prerequisites (libraries and eth1 client), establish a default configuration, and prepare the software for immediate use.
+- By executing `systemctl stop besu`, the Besu client can be gracefully stopped.
+- The option to adjust optional configuration settings for a package can be accessed through `dpkg-reconfigure` geth.
+
+## Included software
+
+Execution clients
+
+    Go-ethereum
```
Comment (taxmeifyoucan) [inline projects/eth-deb-repository.md:65]: OpenEthereum is deprecated for some time now. Maybe can be switched for reth. 

Are you aiming to support all clients? You could define MVP scope with one or two easiest client pairs

Comment (eenagy) [reply projects/eth-deb-repository.md:65]: Yes, I aim for the support for all the clients as binary at least, for configuration, not sure if it's possible in this timeline.

File: projects/eth-deb-repository.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Another perspective to consider is that many development teams lack the resources to adapt their software for various distributions, as this entails a substantial time investment. Having a centralized repository for Ethereum-related packaging, focused solely on the mission of making Ethereum software user-friendly, could greatly alleviate this issue.
+
+Third aspect, that node runners are the most up to date on the topics of what configuration or security settings needs to be adjusted, or what issues a platform might pose. It's good idea to provide a central area where such discussions can be exchanged.
```
[the remark this answers is not in the document]
Comment (eenagy) [reply projects/eth-deb-repository.md:13]: User story downstairs. Please let me know your thoughts.

[the remark this answers is not in the document]
Comment (eenagy) [reply projects/eth-deb-repository.md:13]: Also have many user stories. Not sure if I should include all of them. Here is the one I wrote for my internal use. 

As a node runner
I want to use apt-get to dowload the software
So I can save time and cognitive resources

As a node admin
I want to be offered defaults or configurations
So I can simplify the admin tasks

As a developer
I want to distribute my software much more easily
So I can focus on much more important tasks related to the software I'm developing

As a community member
I want to know where to point people who want to run a node
So I can say running a node is easy

As a security auditor
I want to run tests or audits against the packages and settings
So I can advise for best practices or flag security vulnerabilities or fix security holes by modifying the package (Debian)

[the remark this answers is not in the document]
Comment (taxmeifyoucan) [reply projects/eth-deb-repository.md:13]: I commented before reading the whole thing, this is great!

**Comment by taxmeifyoucan** [comment]
@eenagy thanks for all the updates! Ready to merge?

**Pull request merged**